.gitignore

```diff
@@ -6,4 +6,5 @@
 Homestead.json
 Homestead.yaml
 .env
+composer.lock
```

Comment:
Don't add `composer.lock` to `.gitignore` file. You can push this file to git repo.